src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.de.xlf

```diff
@@ -762,7 +762,7 @@
             </trans-unit>
             <trans-unit id="a50af522de18a35014430b4720e29a4f" resname="form.validation-warning">
                 <source>form.validation-warning</source>
-                <target><![CDATA[Bitte überprüfen deine Eingabe.<br/>Erst wenn alle Felder korrekt ausgefüllt sind, kann gespeichert werden.]]></target>
+                <target><![CDATA[Bitte überprüfe deine Eingabe. Erst wenn alle Felder korrekt ausgefüllt sind, kann gespeichert werden.]]></target>
```

Comment:
Can't you omit the cdata now also? It's a bit strange to only have that on one translation.